travis/test_pylint

```diff
@@ -9,12 +9,54 @@ import os
 
 import run_pylint
 
+from getaddons import get_modules_changed
 
 pylint_rcfile = os.path.join(
     os.path.dirname(os.path.realpath(__file__)),
     'cfg',
     "travis_run_pylint.cfg")
-result = run_pylint.main([
+count_errors = run_pylint.main([
     "--config-file=" + pylint_rcfile,
 ], standalone_mode=False)
-exit(result)
+
+
+pylint_rcfile_pr = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)),
+    'cfg',
+    "travis_run_pylint_pr.cfg")
+
+is_pull_request = os.environ.get(
+    'TRAVIS_PULL_REQUEST', 'false') != 'false'
```

Comment:
Why not stick to the 0/1 values like for other directives? Even better, why not do the check after a string eval to accept both 0/1 and false/true?